README.md

```diff
@@ -733,6 +745,7 @@ Table of Contents
 
    * [airtable.com](https://airtable.com/) — Looks like a spreadsheet, but it's a relational database, unlimited bases, 1,200 rows/base and 1,000 API requests/month
    * [Astra](https://astra.datastax.com/register) — Cloud Native Cassandra as a Service with 10GB free tier
+   * [Azure Cosmos DB](https://azure.microsoft.com/services/cosmos-db/?WT.mc_id=freefordev-github-yolasors) - 5GB storage and 400 RUs of provisioned throughput)
```

Comment:
I prefer to list things once only or companies once only, so with the big block above covering azure, thats the only place we should list azure.

Reply:
Donce